data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java

```diff
@@ -173,21 +151,6 @@ public Executor getAsyncExecutor() {
         }).findFirst().orElse(null);
     }
 
-    @Bean
-    protected WebMvcConfigurer webMvcConfigurer() {
-        return new WebMvcConfigurer() {
-            @Override
-            public void configureAsyncSupport(final AsyncSupportConfigurer configurer) {
-                configurer.setTaskExecutor(getTaskExecutor());
-            }
-        };
-    }
-
-    @Bean(name = "concurrentTaskExecutor")
-    public ConcurrentTaskExecutor getTaskExecutor() {
-        return new ConcurrentTaskExecutor(this.getAsyncExecutor());
-    }
-
```

Comment:
Again is this an intentional deletion or an artefact of a merge going slightly wrong?